tools/testgrid/testgrid.go

```diff
@@ -25,9 +25,9 @@ import (
 )
 
 const (
-	// Default filename to store output that acts as input to testgrid.
+	// Filename to store output that acts as input to testgrid.
 	// Should be of the form junit_*.xml
-	filename = "/junit_knative.xml"
+	Filename = "/junit_knative.xml"
```

Comment:
Removing the / from here, and add it to the relevant places to avoid surprises when using this const.